File: Easy/1. Two Sum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        
        var hash: [Int : Int] = [:]
        var res : [Int] = [];
        
        for (i, n) in nums.enumerated() {
            if let index = hash[target - n]{
                res.append(index)
                res.append(i)
            }
            hash[n] = i 
        }
        return res;
    }
}

File: Easy/1. Two Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        d = {} 
        for index, val in enumerate(nums):
            complement = target - val
            if complement in d:
                return [d[complement], index]
            d[val] = index
        return -1